vxdev/default-env

```diff
@@ -1,3 +1,5 @@
 REACT_APP_VX_DEV=true
-# Uncomment the following line, then run 'Update Code',  to enable write in adjudication in VxDev
-# REACT_APP_VX_ENABLE_WRITE_IN_ADJUDICATION=true
+REACT_APP_VX_ENABLE_WRITE_IN_ADJUDICATION=true
```

Comment:
Is this needed for this PR or did it just get included (fine either way, just double checking)

Reply:
Oh if I'd read your description you said that it is supposed to be here. My bad